src/index.js

```diff
@@ -16,8 +16,8 @@ getData().then((data) => {
             <span>5 likes</span>
           </div>
           <div class="card-footer">
-            <button>Comments</button>
-            <button>Reservations</button>
+            <span><button>Comments<button></span>
```

Comment:
Nice modification @Bateyjosue !
Could you consider adding classes to the Buttons to ease DOM selection?

Reply:
Still waiting for your feature especially the fetch API. Thanks